TeleportMenu.lua

```diff
@@ -212,6 +214,38 @@ function tpm:updateAvailableWormholes()
 	end
 end
 
+function tpm:updateAvailableSeasonalTeleport()
+	local playerFaction = UnitFactionGroup("player")
+	local siegeOfBoralus = -1
+	if playerFaction == "Alliance" then
```

Comment:
Does this return "Aliance" for every locale?

Reply:
Yes:

https://warcraft.wiki.gg/wiki/API_UnitFactionGroup
First value returned is always English, second value is localized.